.gitlab/update-otel-deps.sh

```diff
@@ -15,18 +17,17 @@ cd "${ROOT_DIR}"
 create_collector_pr() {
   local repo="signalfx/splunk-otel-collector"
   local repo_url="https://srv-gh-o11y-gdi:${GITHUB_TOKEN}@github.com/${repo}.git"
-  local update_deps_branch="create-pull-request/update-deps"
-  local message="Update OpenTelemetry Dependencies to latest"
+  local update_deps_branch="$BRANCH"
```

Comment:
Can we rename existing references to `update_deps_branch` to simply use `BRANCH` instead? Unless there's a specific reason that I'm missing, it would make more sense to me to use one variable for everything instead of copying the value here and using a different one.

Reply:
sure, done